tests/manage/mcg/test_multi_region.py

```diff
@@ -122,18 +122,22 @@ def test_multiregion_mirror(
 
         # Verify integrity of B
         # Retrieve all objects from MCG bucket to result dir in Pod
-        sync_object_directory(awscli_pod, mcg_bucket_path, local_temp_path, mcg_obj)
+        sync_object_directory(
```

Comment:
sync_object_directory function is used in certain e2e level tests as well, should be modified there as well?

example: https://github.com/red-hat-storage/ocs-ci/blob/26bd3298a512a596145aa2376e08dd4cd2f84723/tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops_rpc.py#L539

Reply:
`sync_object_directory` function itself did not change so i dont believe you need to change it as well, the new awscli pod image contains the test files of `ocsci-test-files` (which were retrieved for each test) and now said files resides in the `/test_objects/` dir.